prepline_general/api/general.py

```diff
@@ -225,12 +227,16 @@ def partition_pdf_splits(
     results: List[Element] = []
     page_iterator = get_pdf_splits(pdf_pages, split_size=pages_per_pdf)
 
+    # -- retain the original value for the next step of assembling page splits. --
+    unique_element_ids = partition_kwargs.pop("unique_element_ids")
```

Comment:
we don't need this anymore, do we, since we'll get this from unstructured now?

Reply:
that's right, I'm reworking this PR to utilize `starting_page_number` instead